website/versioned_docs/version-v1.3_alpha/integrating.md

```diff
@@ -49,6 +59,59 @@ maci-cli deployPoll \
     --vote-option-tree-depth 2
 ```
 
+### Deploy contracts in `maci/contracts`
+
+This could also be done via running commands in `maci/contracts`. Please download the maci repository, install and build everything, then navigate to `contracts/` folder.
+
+First of all, modify the `deploy-config.json` file:
+
+```
+{
+    "platform": {
+      "ConstantInitialVoiceCreditProxy": {
+        "deploy": true,
+        "amount": 99
+      },
+      "FreeForAllGatekeeper": {
+        "deploy": true
+      },
+      "EASGatekeeper": {
+        "deploy": false,
+        "easAddress": "0x4200000000000000000000000000000000000021",
+        "schema": "0xe2636f31239f7948afdd9a9c477048b7fc2a089c347af60e3aa1251e5bf63e5c",
+        "attester": "attester-address"
+      },
+      "MACI": {
+        "stateTreeDepth": 10,
+        "gatekeeper": "FreeForAllGatekeeper"
+      },
+      "VkRegistry": {
+        "stateTreeDepth": 10,
+        "intStateTreeDepth": 1,
+        "messageTreeDepth": 2,
+        "voteOptionTreeDepth": 2,
+        "messageBatchDepth": 1,
+        "processMessagesZkey": "../cli/zkeys/ProcessMessagesNonQv_10-2-1-2_test/ProcessMessagesNonQv_10-2-1-2_test.0.zkey",
+        "tallyVotesZkey": "../cli/zkeys/TallyVotesNonQv_10-1-2_test/TallyVotesNonQv_10-1-2_test.0.zkey"
+      },
+      "Poll": {
+        "pollDuration": 30,
+        "coordinatorPubkey": "macipk.9a59264310d95cfd8eb7083aebeba221b5c26e77427f12b7c0f50bc1cc35e621",
+        "useQuadraticVoting": true
+      }
+    }
+  }
```

Comment:
This is not the same deploy-config.json as currently in main branch and there is an example in contracts folder for this deploy config.